pingen_env/hooks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+def post_init_hook(cr, registry):
+    # When installing pingen_env we want to delete data from DB
+    cr.execute(
+        """UPDATE res_company
```

Comment:
This seems a solution of your current problem in existing DBs instead of a generic one.

Reply:
@pedrobaeza  updated readme + removed hook, i think we should be ok now